src/histogram.rs

```diff
@@ -309,6 +309,7 @@ mod coarse {
 /// observe.
 ///
 /// NOTICE: A timer can be observed only once (automatically or manually).
+#[must_use]
```

Comment:
I'd like to have some notes.

Reply:
According to https://doc.rust-lang.org/reference/attributes.html#lint-check-attributes we can do `#[must_use = "message"]` and note this. Can we do this? :grin:

Reply:
nice idea!